packages/vscode-extension/src/commonlib/azureLogin.ts

```diff
@@ -339,6 +339,8 @@ export class AzureAccountManager extends login implements AzureAccountProvider {
   async addStatusChangeEvent() {
     const azureAccount: AzureAccount =
       vscode.extensions.getExtension<AzureAccount>("ms-vscode.azure-account")!.exports;
+    // wait for Azure Account extension initialize
+    await azureAccount.waitForSubscriptions();
```

Comment:
This may increase our toolkit activation time... If this is the only place to fix, try a safer way:

``` typescript
private string currentStatus;
...
async addStatusChangeEvent() {
    ...
    this.currentStatus = azureAccount.status;
    azureAccount.onStatusChanged(async (event) => {
        if (event === loggedOut && this.currentStatus !== initializing) {
            this.currentStatus = event;
            ...
        }
    });
}
```

Reply:
Good idea, changed to this behavior.